### PR TITLE
Reformatting annotations of ``Behaviors``

### PR DESCRIPTION
**Pull request by manumafe98**
Related issue: [#5087](https://github.com/MovingBlocks/Terasology/issues/5087)

The goal of this PR is to replace old annotations in ``BehavoirTests`` and ``MovementTests``

### CODE REVIEW

File: src/test/java/org/terasology/module/behaviors/BehaviorTests.java [hunk 1]
```diff
@@ -5,30 +5,25 @@
...
+import org.terasology.engine.integrationenvironment.jupiter.IntegrationEnvironment;
 
 import java.util.Optional;
 import java.util.stream.Stream;
 
 
-@ExtendWith(MTEExtension.class)
-@Dependencies("Behaviors")
-@Tag("MteTest")
+@IntegrationEnvironment(dependencies = {"Behaviors"})
```
Comment (jdrueckert) [inline src/test/java/org/terasology/module/behaviors/BehaviorTests.java]: please remove the curly braces

File: src/test/java/org/terasology/module/behaviors/MovementTests.java [hunk 2]
```diff
@@ -35,14 +31,13 @@
...
+import org.terasology.engine.integrationenvironment.jupiter.IntegrationEnvironment;
 
 import java.util.Arrays;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-@Dependencies("Behaviors")
-@Tag("MteTest")
-@ExtendWith(MTEExtension.class)
+@IntegrationEnvironment(dependencies = {"Behaviors"})
```
Comment (jdrueckert) [inline src/test/java/org/terasology/module/behaviors/MovementTests.java]: same here

**Review by jdrueckert — CHANGES_REQUESTED** [review]

File: src/test/java/org/terasology/module/behaviors/BehaviorTests.java (hunk 1 above)
Comment (manumafe98) [reply src/test/java/org/terasology/module/behaviors/BehaviorTests.java]: Done!

File: src/test/java/org/terasology/module/behaviors/MovementTests.java (hunk 2 above)
Comment (manumafe98) [reply src/test/java/org/terasology/module/behaviors/MovementTests.java]: Done!

**Review by jdrueckert — APPROVED** [review]

**Comment by jdrueckert** [comment]
`Behavior` integration tests being flaky is a known issue. Merging despite unsuccessful checks.

**Pull request merged**